pkg/backend/backend.go

```diff
@@ -189,6 +189,19 @@ type Backend interface {
 	CurrentUser() (string, error)
 }
 
+// SpecificDeploymentExporter is an interface that extends Backend, which adds the ability to export
```

Comment:
This doesn't need to extend `Backend` (it's generally idiomatic to keep interfaces as narrow as possible). LGTM otherwise.

Reply:
Done.